xcode/Subconscious/Shared/Components/AppView.swift

```diff
@@ -298,18 +326,25 @@ enum GatewaySyncStatus: Equatable {
 //  MARK: Model
 struct AppModel: ModelProtocol {
     /// Is Noosphere enabled?
-    /// We assign UserDefault property to model property at startup.
+    ///
+    /// This property is updated at `.start` with the value stored in
+    /// `AppDefaults.standard.isNoosphereEnabled`.
+    ///
     /// This property is changed via `persistNoosphereEnabled`, which will
     /// update both the model property (triggering a view re-render)
     /// and persist the new value to UserDefaults.
-    var isNoosphereEnabled = AppDefaults.standard.isNoosphereEnabled
+    var isNoosphereEnabled = false
```

Comment:
Previously we were loading several initializer defaults directly from `AppDefaults`.

This works, but it means that initializing the model becomes stateful, and this does not play well with tests, since the model's values will be the values of whatever got set previously.

I've moved all of these `AppDefaults` values to `.start`. This brings us one step closer to eventually moving `AppDefaults` into the `AppEnvironment`, allowing us to fully mock the `AppDefaults`.

When we upgrade `ObservableStore`, the new factory initializer will help us do even better here. It lets us generate the first state and fx from a special update function, allowing for this kind of "loading from IO" without compromising on mockability and testability https://github.com/subconsciousnetwork/ObservableStore/pull/30